docs/manual/signature-checker.tex

```diff
@@ -14,7 +14,10 @@ \chapter{Signature String Checker for string representations of types\label{sign
 
 \section{Signature annotations\label{signature-annotations}}
 
-Java defines six formats for the string representation of a type.
+\subsection{String representation of types\label{signature-annotations-types}}
+
+Java defines seven formats for the string representation of a type.
```

Comment:
I adjusted the text to reflect my understanding of the number of formats.

Should `@JavaTypeSignature` be listed as a seventh representation for types, both in the list in section 15.1 "Signature annotations" and also in the table in that section?  (I think so.)
I think that will clarify the relationship between Java type signatures and the other representations of Java types.

Reply:
It might not be possible to add another column for `JavaTypeSignature`s since they're the same as `FieldDescriptor`s for non generic types. I've added brief descriptions of these types to section 15.1.2.